### PR TITLE
added test from task 8 / task 9 and relevant pages

### PR DESCRIPTION
**Pull request by ashchavinska**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/components/ParagraphContent.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+public class ParagraphContent extends AbstractUIObject {
+
+    @FindBy(xpath="./a")
+    public List<ExtendedWebElement> title;
+
+    public ParagraphContent(WebDriver driver, SearchContext sc) {
+        super(driver, sc);
+    }
+
+    public List<ExtendedWebElement> getTitle() {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/components/ParagraphContent.java]: return String

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/GlossaryPage.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    public GlossaryPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public boolean isGlossaryPageOpen() {
+        return title.isPresent() && title.getText().equals("Mobile terms glossary");
+    }
+
+    public List<ExtendedWebElement> getParagraphTitles() {
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/pages/GlossaryPage.java]: see above

File: src/main/java/com/qaprosoft/carina/demo/gui/components/ParagraphContent.java (hunk 1 above)
Comment (ashchavinska) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/ParagraphContent.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/pages/GlossaryPage.java (hunk 2 above)
Comment (ashchavinska) [reply src/main/java/com/qaprosoft/carina/demo/gui/pages/GlossaryPage.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 3]
```diff
@@ -142,4 +142,33 @@ public void verifySearchingProcess() {
...
+        List<ParagraphContent> contents = glossaryPage.getParagraphContents();
+        Assert.assertTrue(titles.size()==contents.size(), "Size doesn't match");
+
+        for (int i = 0; i < titles.size(); i++) {
+            String title = titles.get(i);
+            List<String> content = contents.get(i).getElements();
+            for (String item : content) {
+                char itemFirstChar = item.charAt(0);
+                if (title.equals("0 - 9")) {
+                    Assert.assertTrue(Character.isDigit(itemFirstChar), "First char doesn't match");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: wrong message

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java [hunk 4]
```diff
@@ -142,4 +142,33 @@ public void verifySearchingProcess() {
...
+                }
+                else {
+                    Assert.assertEquals(Character.toLowerCase(itemFirstChar), Character.toLowerCase(title.charAt(0)), "First char doesn't match");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: wrong message

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 3 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

File: src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java (hunk 4 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/WebArenaTest.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**